libs/payments/cart/src/lib/cart.error.ts

```diff
@@ -22,24 +22,32 @@ export class CartError extends BaseError {
       cause,
       info,
     });
+    this.name = 'CartError';
+    Object.setPrototypeOf(this, CartError.prototype);
```

Comment:
Just curious and for logging purposes, why is this necessary?

Reply:
Object.setPrototype is used for instanceof checks, which comes into play for several of our tests. instanceof check are inconsistent otherwise